src/NzbDrone.Core.Test/Download/DownloadClientTests/TransmissionTests/TransmissionFixture.cs

```diff
@@ -148,8 +148,8 @@ public void Download_should_get_hash_from_magnet_url(string magnetUrl, string ex
         [TestCase(TransmissionTorrentStatus.Check, DownloadItemStatus.Downloading)]
         [TestCase(TransmissionTorrentStatus.Queued, DownloadItemStatus.Queued)]
         [TestCase(TransmissionTorrentStatus.Downloading, DownloadItemStatus.Downloading)]
-        [TestCase(TransmissionTorrentStatus.SeedingWait, DownloadItemStatus.Completed)]
```

Comment:
Should seeding torrents still be marked as downloading rather than completed? Also happens in parent commit.

Reply:
Let me look into this more. TransmissionBase seems to disagree with this test.

Reply:
In this test case it’s looking at a torrent that has `leftuntildone` Set to 1000. Downloads are not marked complete until `leftuntildone` is 0 and status is seeding. Thus test should produce a downloading status in this case and not complete.